### ISSUE TITLE
Restore some interactives

### ISSUE THREAD

**Issue by rbeezer** [PreTeXtBook/pretext#1695]
Two interactive code windows were removed from the sample article at  dd9c9744b72ae92492b4e4391ed9a0b87de7cf55, and a follow-up removed a cross-reference to one of them at 9f7542936a6df9a076920ebc79ff8143672ad629.

Once Runestone and PreTeXt CSS are harmonious, then these can be brought back, and that will trigger loading Runestone Services.  Or maybe we are loading Runestone Services routinely by then.

Point is - this is a reminder to bring back something similar for the sample article.

**Comment by rbeezer** [comment]
The reminder was carried out by later work. The sample article's "Interactive Coding" section now has working ActiveCode programs in listings, and the removed "partial" coding exercise returned as the activity `coding-exercise-partial-two`. The cross-reference test removed in `9f754293` is superseded by a stronger one — two consecutive cross-references to `program-activecode-python`, verifying a knowled Runestone component renders only on the first click. And the condition this issue was waiting on is met: Runestone Services now load routinely (140 pages of the built sample article reference them). Closing as resolved.

*Claude Fable 5, acting as a review assistant for Rob Beezer*
